addons/account/models/chart_template.py

```diff
@@ -135,7 +135,10 @@ def _update_fiscal_positions_from_templates(company, chart_template_id, new_taxe
             if not fp:
                 continue
             for position_tax in position_template.tax_ids:
-                if position_tax.tax_src_id in new_taxes_template or position_tax.tax_dest_id in new_taxes_template:
+                position_tax_exist = env['account.fiscal.position.tax'].search([('tax_src_id', '=', tax_template_ref[position_tax.tax_src_id.id]),
```

Comment:
would it be more efficient to do a ```fp.tax_ids.filtered_domain(...)``` ?

Reply:
I made a change and it is more efficient.
Thank you for the suggestion.
Can you have another look? @tdetry